.github/actions/wasi/wasm32/action.yml

```diff
@@ -0,0 +1,12 @@
+name: "Ubuntu wasm32 dependencies"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Download wasm32 libclang runtime
+      shell: bash
+      run: |
+        sudo mkdir -p "$(clang --print-resource-dir)"
+        sudo wget -c \
+          https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-21/libclang_rt.builtins-wasm32-wasi-21.0.tar.gz \
```

Comment:
Don't we download it via `wasi-sdk_bootstrap.cmake`?

Reply:
> Don't we download it via `wasi-sdk_bootstrap.cmake`?

Yep! Thanks for the notice. I've removed this unnecessary step 👍